docs/docs/support/advisory/a10007.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Technical Advisory 10007
+---
+
+## Date and Version
+
+Version: Upcoming
+
+Date: Upcoming
+
+## Description
+
+In upcoming ZITADEL versions, RBAC also applies to [system users defined in the ZITADEL runtime configuration](/guides/integrate/access-zitadel-system-api#runtime-configuration).
+This enables fine grained access control to the system API as well as other APIs for system users.
+ZITADEL defines the new default roles *SYSTEM_OWNER* and *SYSTEM_OWNER_VIEWER*.
+System users without any memberships defined in the configuration will be assigned the *SYSTEM_OWNER* role.
+**Self-hosting users who define their own custom mapping at the InternalAuthZ.RolePermissionMappings configuration section**, have to define the *SYSTEM_OWNER* role in their configuration too to be able to access the system API with the default system user membership.
+
+## Statement
+
+This change is tracked in the following PR: [feat: add SYSTEM_OWNER role](https://github.com/zitadel/zitadel/pull/6765).
+As soon as the release version is published, we will include the version here.
+
+## Mitigation
+
+If you have a custom role mapping configured, make sure you configure the new role *SYSTEM_OWNER* before migrating to upcoming ZITADEL versions.
+As a reference, these are the default mappings:
```

Comment:
Am I right that those who have no custom mapping, do not have to take any action? Maybe we should clearly state that.

Reply:
added it in a new header